samples/FunctionApp/FunctionApp.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.0.1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.0.3" OutputItemType="Analyzer" />
```

Comment:
Should this be[ 1.0.4](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Sdk/1.0.4) which seems to be the latest ?

Reply:
The latest is now 1.3.0